src/types/formTypes.ts

```diff
@@ -7,6 +7,18 @@ import {
 import { Five, One } from 'utils/constants';
 import { z } from 'zod';
 
+export enum FormFieldsHeaderError {
+  firstName = 'Enter your first name',
+  lastName = 'Enter your last name',
+  email = 'Invalid email address',
+  financialInstitutions = 'Select the institution for which you are authorized to file',
+  tin = 'Enter your Federal Taxpayer Identification Number (TIN)',
+  name = "Enter your financial institution's name",
+  lei = "Enter your financial institution's Legal Entity Identifier (LEI)",
+  sbl_institution_types = 'Select your type of financial institution',
+  sbl_institution_types_other = 'Enter a type of financial institution for "Other"',
+}
```

Comment:
My local git history says this was added 9mo ago.  Don't see any references to it elsewhere in code, so probably safe to delete.